src/OpenTelemetry.Instrumentation.ConfluentKafka/CHANGELOG.md

```diff
@@ -1,5 +1,7 @@
 # Changelog
 
+- Add `ConsumeAndProcessMessageAsync` extension method to `ConsumeResult` to ease connecting traces.
+
 ## Unreleased
```

Comment:
I do not think that it is worth to add changelog. It will be part of the initial release.

```suggestion
## Unreleased
```